adafruit_ble_file_transfer.py

```diff
@@ -47,16 +47,21 @@
 class FileTransferUUID(VendorUUID):
     """UUIDs with the CircuitPython base UUID."""
 
+    # pylint: disable=too-few-public-methods
+
     def __init__(self, uuid16):
-        uuid128 = bytearray("!refsnaTeliF".encode("utf-8") + b"\x00\x00\xaf\xad")
+        uuid128 = bytearray("refsnarTeliF".encode("utf-8") + b"\x00\x00\xaf\xad")
```

Comment:
`b"refsnarTeliF"` ?

Reply:
"FileTransfer" backwards

Reply:
Right, but why not use a `b""` bytes string instead of this method of getting one?

Reply:
I could but the encode allows someone to use a different utf-8 character if desired. It's backwards because some clients show the uuid in reverse order from this.